src/utils/aggregationUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { AggregatedPosition } from '../types/aggregation';
+import { LiquidityChanges } from '../types/liquidity';
+
+export function aggregatePositions(userPoolPositions: LiquidityChanges[]): AggregatedPosition[] {
+  const aggregatedPositions: AggregatedPosition[] = [];
+
+  userPoolPositions.forEach((position) => {
+    const existingPosition = aggregatedPositions.find((p) => p.poolId === position.pool.poolIdx);
+
+    if (existingPosition) {
+      existingPosition.baseAmount += parseFloat(position.baseFlow);
+      existingPosition.quoteAmount += parseFloat(position.quoteFlow);
```

Comment:
I think it's better to use `bignumber` from `mathjs` library to add all numbers, to prevent floating errors.